### ISSUE TITLE
Few questions.

### ISSUE THREAD

**Issue by ghost** [biscuitlang/bl#110]
@travisdoor Hi, I want to ask a few questions about BL programming Language:

**1.** Where can I find stdio library for BL with `printf`, `fprintf` `sprintf`,`scanf` and etc.

**2.** How to read input? (like in C `scanf`?)

**3.** How to declare namespaces?

**4.** Where can I find documentation for it?

**5.** How to use `for`/`repeat` loop in BL?

**6.** Is there something like `switch`?

**7.** What about catching exceptions in BL? (`try-catch`?)

**Comment by travisdoor** [comment]
Hi, documentation for BL can be found [here](http://biscuitlang.org).

1) BL does not use those functions at all, you can use [print](http://biscuitlang.org/versions/0.7.2/api/std/out-44.html?highlight=print) function for printing to stdout declared in `lib/bl/api/std/print.bl`.

2) API for this is not implemented yet, I work on this project only in my spare time so I basically implement only things I need. 
Feel free to add implementation for this. 
You can still use libc functions like this: `scanf :: fn (format: *u8, arg: *u8) s32 #extern;` but BL vargs is not compatible with C due to different implementation, so usage is quite limited.

3) There are no namespaces in language, but I have plan to support them in future. It will be something like file-scoped module name.

4) [here](http://biscuitlang.org)

5) [example](http://biscuitlang.org/versions/0.7.2/examples/003_flow_control.bl.html)

6) Yes, switch is there, but documentation is missing. I'll add it soon. 
```
main :: fn () s32 {
    i :: 1;
    switch i {
        0 { print("Zero"); }
        1 { print("One"); }
        default { print("Other"); }
    }
    return 0;
}
```

7) Exceptions are not supported and there is no plan to support them, you can use return codes. However multiple return values will be supported in future so it will be possible to return value and error code at once.

**Comment by ghost** [comment]
@travisdoor Will you fix this in a free time? (#113 #116)

**Comment by travisdoor** [comment]
Fixed in commit https://github.com/travisdoor/bl/commit/aff2b577d57c21745ecb26686603093df8b401e1
